### DISCUSSION TITLE
what distros are supported

### DISCUSSION THREAD

**Discussion by unicornsandcandies** [jc141x/jc141-bash#27] [General]
I see this changing every second. I use Fedora. Can I use this releases there? Because I could previously. What is happening here exactly? 

Can the authors at least provide something like an ready to use podman file for distrobox? Or basically anything at this point. Just so I'm not wasting my time when suddenly support for my distro is dropped and I as a non engaged user have no idea what's happening

**Comment by adminx01** [comment]
we are not able to update the dwarfs package for fedora anymore. and it is required to have the latest version in order to run a lot of releases.

its not in our interest to say something is supported when the reality is not so.

you could use distrobox with arch and get the dwarfs package from AUR easily.
